### ISSUE TITLE
How to get librain to work with Vulcan

### ISSUE THREAD

**Issue by BadPlane** [skiselkov/librain#32]
Hey, it seems like people have librain working with Vulcan and metal, but it still doesn't seem to work. Is there a clear yes or no to this question or is there a tutorial to get it to work?

**Comment by crazytimtimtim** [comment]
Librain does _not_ work with Vulkan. Can you explain the people who got it to work?

**Comment by BadPlane** [comment]
Ok, I thought I saw some people saying that Librain supported Vulcan now. Is there an ETA on Vulcan support

**Comment by crazytimtimtim** [comment]
You'll need to ask @skiselkov about that.

Anyway, you may think Zibo supports it, but it actually doesn't. There are three rain effect options in the EFB. One uses Librain, the other two are basically just images that it overlays on the windshield. That's why the first option doesn't work in Vulkan.

**Comment by BadPlane** [comment]
Ok, so I just have to wait for librarian to get support, thx

**Comment by crazytimtimtim** [comment]
Yeah, no problem! I wonder if skiselkov has just abandoned it... Vulkan's been out for a while now.

**Comment by BadPlane** [comment]
Well, he's been summoned now, so we can see what he says

**Comment by crazytimtimtim** [comment]
Nothing...

**Comment by FGremlin** [comment]
Hi @skiselkov are you planning to release the Vulcan version? I know, your TBM 900 has the rain effects and it's a beautiful plane, but can we get Librain with Vulcan support for other developers. Thank you!

**Comment by crazytimtimtim** [comment]
He's active on github, just did some stuff 2 days ago... not sure why he isn't responding.

@skiselkov ?

**Comment by OVK-Az3rky** [comment]
bruh its been 2 years 💀  @skiselkov  ??????
